src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -173,7 +213,7 @@ class StacktraceLink extends AsyncComponent<Props, State> {
                 deps =>
                   this.project && (
                     <StacktraceLinkModal
-                      onSubmit={() => this.handleSubmit()}
+                      onSubmit={() => this.handleSubmit}
```

Comment:
@mgaeta hmmm this seems strange, are we actually invoking `handleSubmit` here? I'm also confused why changing stack trace linking is part of a PR about  the warnings caused by`getEndPointString`.